packages/components/src/Components/InsightsLabel/CriticalIcon.js

```diff
@@ -0,0 +1,11 @@
+import './critical-icon.scss';
+
+import React from 'react';
+
+const CriticalIcon = () => <React.Fragment>
```

Comment:
Do we need to wrap it in `Fragment`? Since it's just a `svg` I think we are good without it.